test/index.js

```diff
@@ -0,0 +1,41 @@
+var R = require('..');
+var eq = require('./shared/eq');
+var fs = require('fs');
+var path = require('path');
+
+
+var sourceMethods = function(dir) {
+  var isJsFile = function(file) { return file.match(/\.js$/); }
+  var removeJsEnding = function(file) { return file.replace('.js', ''); }
+  return fs.readdirSync(dir).filter(isJsFile).map(removeJsEnding);
+}
+
+/**
+ * Convention is
+ *  * Actual API—all `./src/*.js` files are top level API methods
+ *  * Exported API—object in `./index.js` to be exported
+ *  * Actual and exported APIs should be the same
+ *
+ * Two cases, when exported and actual APIs might differ
+ *  1. newly added API `./src/method.js` is forgotten to be added into './index.js'
+ *  2. API method is deprecated and actual source file from `./src/` removed,
+ *     while continues to exist in `./index.js`
+ *
+ * 1st case is detected in first assertion, and detailed in second one
+ *
+ * 2st case doesnt need detection, because NodeJS will throw an error
+ * if you would attempt to require non existing file
+ */
+describe('API surface', function() {
+  var exported = Object.keys(R);
+  var actual = sourceMethods(path.join(__dirname, '..', 'src'));
```

Comment:
Previosly we sorted these arrays, and then just `eq`ed them. While its simple, it makes it hard to identify problem parts, when arrays are 200+ items each.

I would suggest to go with two checks. First `length`s check is sufficient. And second check is human friendly. Check it out:  
![screen shot 2016-10-25 at 12 58 35 am](https://cloud.githubusercontent.com/assets/559321/19667602/15a60722-9a52-11e6-8d08-862ae67a51f9.png)